tests/testPersistance.py

```diff
@@ -0,0 +1,119 @@
+"""Tests that read and write exposures and dcr models to disk."""
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function, division, absolute_import
+
+from itertools import izip
+import unittest
+
+import lsst.utils.tests
+
+from python.generateTemplate import GenerateTemplate
+from python.test_utils import DcrModelTestBase
+
+
+class PersistanceTestCase(DcrModelTestBase, lsst.utils.tests.TestCase):
+    """Tests that read and write exposures and dcr models to disk."""
+
+    def test_create_exposure(self):
+        """Test that the data retrieved from an exposure is the same as what it was initialized with.
+
+        Raises
+        ------
+        Exception
+            Any exception is collected and passed up to the caller
+        """
+        self.assertFloatsAlmostEqual(self.exposure.getMaskedImage().getImage().getArray(), self.array)
+        meta = self.exposure.getMetadata()
+        # Check that the required metadata is present:
+        try:
+            meta.get("ZENITH")
+        except Exception as e:
+            raise e
```

Comment:
Consider  replacing these 4 lines with`self.assertTrue(meta.exists("ZENITH")). Or if you know what value to expect, it would be even better to assume the keyword will be present and check the value.

Also if you ever do want to fail a test in this way I suggest you call `self.fail(...)` instead of `raise e` with a suitable error message as the argument. That gives the desired error report (other kinds of exceptions produce a different error report that looks like a bug in your code).